billing/src/main/java/com/appcoins/wallet/billing/AppcoinsBillingReceiverActivity.kt

```diff
@@ -99,10 +99,18 @@ class AppcoinsBillingReceiverActivity : MessageProcessorActivity() {
     }
   }
 
-  private fun consumePurchase(packageName: String, purchaseToken: String?): Parcelable {
+  private fun consumePurchase(apiVersion: Int, packageName: String,
+                              purchaseToken: String?): Parcelable {
+    if (apiVersion != SUPPORTED_API_VERSION) {
+      val response = Bundle()
+      response.putInt("RESULT_VALUE",
+          AppcoinsBillingBinder.RESULT_BILLING_UNAVAILABLE)
+      return response
+    }
+
     val result = Bundle()
     if (purchaseToken.isNullOrBlank()) {
-      result.putInt(AppcoinsBillingBinder.RESPONSE_CODE,
+      result.putInt("RESULT_VALUE",
```

Comment:
Should we use a macro for this value, since this is used in many cases.

Reply:
you are right, fixed.